localization/languages.json

```diff
@@ -6,5 +6,6 @@
     "it": ["Italian"],
     "id": ["Bahasa Indonesia"],
     "uk": "ru",
-    "be": "ru"
+    "be": "ru",
+    "pl": ["Polski"]
```

Comment:
Please add Polish right after Indonesian, not at the end of the file

Reply:
Done